packages/web-react/DEPRECATIONS-v2.md

```diff
@@ -4,7 +4,187 @@ Following deprecations will be removed in version 2 of the _spirit-web-react_ pa
 
 > Please follow these steps to safely upgrade your design system to Spirit Design System v2 components.
 
-## Add the `danger` icon to your project's assets
+[What are deprecations?][readme-deprecations]
+
+## Deprecations
+
+### Required `id` prop for form components
+
+The `id` prop will be required for all form components.
+
+Related components:
+
+- `Checkbox`
+- `FieldGroup`
+- `FileUploader`
+- `Radio`
+- `Select`
+- `TextArea`
+- `TextField`
+
+#### Migration Guide
+
+Add `id` prop to the form components.
+
+### FileUploaderAttachment `buttonLabel` and `editButtonLabel` props
```

Comment:
We have a codemod for this change now. Should you please add a link to this codemod or an example of how to run it?

Reply:
I think a good example is here: https://mui.com/material-ui/migration/migration-v4/#link-underline-hover